### PR TITLE
Update `tkinter.Misc` for 3.13

### PR DESCRIPTION
**Pull request by max-muoto**
Add new methods on `tkinter.Misc` for 3.13.

### CODE REVIEW

File: stdlib/tkinter/__init__.pyi [hunk 1]
```diff
@@ -319,6 +319,9 @@ class _GridIndexInfo(TypedDict, total=False):
     uniform: str | None
     weight: int
 
+class _BusyInfo(TypedDict):
+    cursor: _Cursor
```
Comment (max-muoto) [inline stdlib/tkinter/__init__.pyi:323]: Doing this to appease stubtest.

**Comment by max-muoto** [comment]
CC: @Akuli

File: stdlib/tkinter/__init__.pyi (hunk 1 above)
Comment (JelleZijlstra) [reply stdlib/tkinter/__init__.pyi:323]: I think I'd prefer to spell out the kwargs explicitly and ignore stubtest; it probably gives a better experience for users calling this function.

File: stdlib/tkinter/__init__.pyi [hunk 2]
```diff
@@ -352,7 +355,29 @@ class Misc:
...
     def bell(self, displayof: Literal[0] | Misc | None = 0) -> None: ...
+    if sys.version_info >= (3, 13):
+        # Supports options from `_BusyInfo``
+        def tk_busy_cget(self, option: Literal["cursor"]) -> _Cursor: ...
+        busy_cget = tk_busy_cget
+        def tk_busy_configure(self, cnf: Any = None, **kw: Any) -> Any: ...
+        tk_busy_config = tk_busy_configure
+        busy_configure = tk_busy_configure
+        busy_config = tk_busy_configure
+        def tk_busy_current(self, pattern: str | None = None) -> list[Widget]: ...
```
Comment (Akuli) [inline stdlib/tkinter/__init__.pyi:370]: Tkinter's `Widget` is misleadingly named. For this type annotation, you want to say "any widget whatsoever", and to spell that, you actually need `Misc`, not `Widget`.

A `Widget` is something you can add inside another widget, i.e. anything except `Tk` and `Toplevel`. But a `Tk` or `Toplevel` can be marked as busy.

```suggestion
        def tk_busy_current(self, pattern: str | None = None) -> list[Misc]: ...
```

File: stdlib/tkinter/__init__.pyi [hunk 3]
```diff
@@ -352,7 +355,29 @@ class Misc:
...
     def after_cancel(self, id: str) -> None: ...
+    if sys.version_info >= (3, 13):
+        def after_info(self, id: str | None = None) -> tuple[str, ...]: ...
+
     def bell(self, displayof: Literal[0] | Misc | None = 0) -> None: ...
+    if sys.version_info >= (3, 13):
+        # Supports options from `_BusyInfo``
+        def tk_busy_cget(self, option: Literal["cursor"]) -> _Cursor: ...
+        busy_cget = tk_busy_cget
+        def tk_busy_configure(self, cnf: Any = None, **kw: Any) -> Any: ...
```
Comment (Akuli) [inline stdlib/tkinter/__init__.pyi:366]: You can overload this similarly to other `configure` methods to get rid of `**kw: Any`. Look at `Tk.configure` for a relatively simple example.

After writing the types for those other `configure` methods, I have realized that there are better ways to do it, but that's beyond the scope of this PR and it's fine to go with the existing pattern. By far the most common use case is passing keyword arguments and ignoring the return value, e.g. `widget.busy_config(cursor="hand2")` in this case.

**Pull request merged**